### ISSUE TITLE
Quest The Dead Fields is broken

### ISSUE THREAD

**Issue by vgroesbeck** [chromiecraft/chromiecraft#5650]
### What client do you play on?

enUS

### Faction

Horde

### Content Phase:

1-19

### Current Behaviour

The quest "The Dead Fields" does not give credit for one of the stages "Enter the Dead Fields." I completed the other stage by killing the named mob (three times), ran over every inch of the field, left and re-entered from every angle, even abandoned and picked it up again, but no matter what I could not get credit for entering the field, so the quest would not complete. 

### Expected Blizzlike Behaviour

I should get credit for entering the field as soon as I enter the field.

### Source

I am not sure what is mean by source. I tried to complete the quest twice, in several different ways, but it would not complete. World chat told me to submit a bug report.

### Steps to reproduce the problem

Be level 14 Horde
Pick up the quest "The Dead Fields" at the Sepulcher in Silverpine Forest.
Run north to the Dead Fields and try to complete the quest by entering the dead fields and killing the named mob in the center of the fields (Nightlash) and collecting her essence.
You can complete all stages of the quest but will not get credit for "entering the field" so the quest will not complete.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/8224a90a99cc40ff9a7a9a3ab4d7aecc40aacd3d

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by elthehablo** [comment]
We tend to base content on how it is on retail/wotlk. So if you claim something is different to how it is on retail, it's considered good practice to supply a source (a video, image (depending on issue), or even better: sniffs). These should show something that you explain in "Expected Blizzlike Behaviour". Thanks for the report. I will take a look tomorrow

**Comment by elthehablo** [comment]
Looks broken indeed. But looking at classic footage it was kind of hard to see when the trigger happens exactly lol

**Comment by asdff134242** [comment]
Duplicate of #5472.

**Comment by 7Katta** [comment]
Came here to report and see that others have reported. Having the same exact issue, cannot trigger completion for "Enter the Dead Fields"- have killed Nightlash numerous times.  Am playing BM Hunter, if that has any relevance.
